packages/editor/src/components/block-actions/index.js

```diff
@@ -79,6 +80,9 @@ export default compose( [
 				}
 
 				const clonedBlocks = blocks.map( ( block ) => cloneBlock( block ) );
+
+				clearSelectedBlock();
```

Comment:
I think we shouldn't need to explicitly clear here. What's causing the new blocks to become selected? Is it `insertBlocks` ? Thinking the editor's reducer for tracking selection should be responding to the insertion, rather than needing to be explicit with clearing the block here.